Source/WSTagsField.swift

```diff
@@ -435,7 +463,18 @@ open class WSTagsField: UIScrollView {
         let text = self.textField.text?.trimmingCharacters(in: CharacterSet.whitespaces) ?? ""
         if text.isEmpty == false && (onVerifyTag?(self, text) ?? true) {
             let tag = WSTag(text)
+            if self.tags.contains(tag) {
+                self.textField.text = ""
+                return nil
+            }
+            
             addTag(tag)
+            if let tagView = tagViews.last, autoSelectTagWhenAdded {
+                //There's a bug that causes the text to be truncated during animation ("New York" becomes "New Y..."). This delays animating the TagView until after it's set in the TextField.
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.15) {
```

Comment:
Hmm, I think we should do another approach. This could not always solve the problem.